### ISSUE TITLE
Feature proposal : ability to rollback, loosing a life

### ISSUE THREAD

**Issue by borisschapira** [esviji/esviji#84]
Sometimes you've planned it all, and suddenly you miss the right line because of your f...... fat fingers. I hate to those times.
I'll loose a life or two to roll back and not be frustrated by this ;)

**Comment by sylzys** [comment]
You could use this feature as a "joker", like making it available once every game. Besides "big finger" misclicks, it could be a real value thorough the game :)

**Comment by borisschapira** [comment]
I still hope for this feature to come, not as a joker. In fact, a game session is so long that sometimes, I pause and come back to finish. I often loose the context and my first few move are quite poor. The ability to get back can help find a better solution. I'm still 100% confident it would help the players :)

**Comment by nhoizey** [comment]
Would you see this as a new action in the pause screen?

**Comment by borisschapira** [comment]
No, I would offer this on the main screen... but I'm not sure where. A new challenge for the responsiveness :+1:

**Comment by nhoizey** [comment]
There are already a few things I'm not happy with in some screen ratios, do you want me to have a seizure? :-p
